src/App.res

```diff
@@ -25,6 +25,8 @@ let make = () => {
         <FullScreenDivDriver />
       </div>
     | "qrData" => <QRCodeDisplay />
+    | "3dsAuth" => <ThreeDSAuth />
+    | "3ds" => <ThreeDSMethod />
```

Comment:
why does `fullscreenMode` have to be string? why not variant?

Reply:
we open these iframe from the hyperloader by sending the postmessage on top of our main iframe by setting these as query param 
once the iframe are loaded we extract the mode and then use them to render the respective modal 

after extracting we can convert them to variants but is it really necessary?